src/main/java/com/getindata/kafka/connect/iceberg/sink/IcebergChangeEvent.java

```diff
@@ -189,7 +193,7 @@ private Object jsonValToIcebergVal(Types.NestedField field,
           val = OffsetDateTime.parse(node.asText());
         }
         else if (node.isNumber()) {
-          Instant instant = Instant.ofEpochSecond(0L, node.asLong() * 1000);
+          Instant instant = Instant.ofEpochMilli(node.asLong());
```

Comment:
I believe that this is database implementation dependent.

Reply:
The change will break on PostgresQL, for example.  Perhaps temporal types need to be divided into specific debezium sink targets?